src/stravalib/tests/conftest.py

```diff
@@ -18,3 +21,26 @@ def mock_strava_api():
 @pytest.fixture
 def client():
     return Client()
+
+
+@pytest.fixture
+def apiv3_instance():
+    """Fixture to create an ApiV3 instance for testing."""
+
+    return ApiV3(
+        access_token="dummy_access_token",
+        client_id=123456,
+        client_secret="clientfoosecret",
+        refresh_token="refresh_value123",
+    )
+
+
+@pytest.fixture
+def mock_strava_env():
```

Comment:
vscode will STORE environment variables like this that are mocked. This was breaking things and I couldn't figure out why. This fixture allows us to mock that piece by just calling it.